ecommerce-ui/src/App.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Usually each Component would have its own test, but I see you understand the principle.